mvrx/src/main/kotlin/com/airbnb/mvrx/Async.kt

```diff
@@ -36,17 +36,13 @@ sealed class Async<out T>(val complete: Boolean, val shouldLoad: Boolean) {
     }
 }
 
-object Uninitialized : Async<Nothing>(complete = false, shouldLoad = true), Incomplete
+object Uninitialized : Async<Nothing>(complete = false, shouldLoad = true, value = null), Incomplete
 
-class Loading<out T> : Async<T>(complete = false, shouldLoad = false), Incomplete {
-    override fun equals(other: Any?) = other is Loading<*>
+data class Loading<out T>(private val value: T? = null) : Async<T>(complete = false, shouldLoad = false, value = value), Incomplete
```

Comment:
Can you update the javadoc to include information as to why you would use value?

Reply:
Sure, I updated the doc from `Async.invoke()` ... do you think it is enough?